stubs/antlr4-python3-runtime/antlr4/BufferedTokenStream.pyi

```diff
@@ -25,15 +25,15 @@ class BufferedTokenStream(TokenStream):
     def LA(self, i: int) -> int: ...
     def LB(self, k: int) -> Token | None: ...
     def LT(self, k: int) -> Token | None: ...
-    def adjustSeekIndex(self, i: int): ...
+    def adjustSeekIndex(self, i: int) -> Incomplete: ...
     def lazyInit(self) -> None: ...
     def setup(self) -> None: ...
-    def setTokenSource(self, tokenSource: Lexer): ...
-    def nextTokenOnChannel(self, i: int, channel: int): ...
-    def previousTokenOnChannel(self, i: int, channel: int): ...
-    def getHiddenTokensToRight(self, tokenIndex: int, channel: int = ...): ...
-    def getHiddenTokensToLeft(self, tokenIndex: int, channel: int = ...): ...
-    def filterForChannel(self, left: int, right: int, channel: int): ...
-    def getSourceName(self): ...
-    def getText(self, start: int = ..., stop: int = ...): ...
+    def setTokenSource(self, tokenSource: Lexer) -> Incomplete: ...
+    def nextTokenOnChannel(self, i: int, channel: int) -> Incomplete: ...
+    def previousTokenOnChannel(self, i: int, channel: int) -> Incomplete: ...
+    def getHiddenTokensToRight(self, tokenIndex: int, channel: int = ...) -> Incomplete: ...
+    def getHiddenTokensToLeft(self, tokenIndex: int, channel: int = ...) -> Incomplete: ...
+    def filterForChannel(self, left: int, right: int, channel: int) -> Incomplete: ...
+    def getSourceName(self) -> Incomplete: ...
+    def getText(self, start: int = ..., stop: int = ...) -> Incomplete: ...
```

Comment:
Apologies for not saying this earlier -- rather than adding `-> Incomplete` in return types everywhere, we generally prefer to add entries to this list here, so that the directory is excluded from pyright's stricter settings. So for a type that's completely unknown, we'd just omit the annotations (if the stub is excluded from pyright's stricter settings):

```py
def foo(x: int, y): ...
```

But if the type is _partially_ known, we'd generally use `Incomplete`:

```py
def foo(x: int, y: list[Incomplete]) -> set[Incomplete]: ...
```